docs/dev_sliding_dot_product.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        As a bonus, it tests their performance too when <code>test_output</code> is set to True.
</pre><p>should be replaced with:</p><pre class="ql-syntax" spellcheck="false">As a bonus, it tests their outputs too when <code>test_output</code> is set to True.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>